config/webpack.config.js

```diff
@@ -6,6 +6,8 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer')
 const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
 const webpack = require('webpack');
 const path = require('path');
+const ENVIRONMENTS = require('../src/site/constants/environments');
+const BUCKETS = require('../src/site/constants/buckets');
```

Comment:
I was hesitant about importing from `src` into `config`, but I preferred the explicitness over something like deriving  build arguments from where our Metalsmith build calls `configGenerator`.